frontend/packages/employee-frontend/src/App.tsx

```diff
@@ -309,6 +310,14 @@ export default function App() {
                 title={i18n.titles.reports}
               />
             )}
+            {featureFlags.voucherValueDecisionsPage && (
+              <RouteWithTitle
+                exact
+                path="/reports/voucher-service-provider-unit/:unitId"
```

Comment:
I think this path should match the unit report path. So they either should be 
`/reports/voucher-service-providers` and `/reports/voucher-service-providers/:unitId` 
or 
`/reports/voucher-service-provider-unit` and `/reports/voucher-service-provider-unit/:unitId`.